using_images/xpaas_images/a_mq.adoc

```diff
@@ -5,4 +5,92 @@
 :icons:
 :experimental:
 
-Information on the Red Hat JBoss A-MQ xPaaS image will be added soon.
+Red Hat JBoss A-MQ (JBoss A-MQ) is available as a containerized xPaaS image that is designed for use with OpenShift v3. It allows developers to quickly deploy an A-MQ message broker in a hybrid cloud environment.
+
+[IMPORTANT]
+There are significant differences in supported configurations and functionality in the JBoss A-MQ image compared to the regular release of JBoss A-MQ.
+
+This topic details the differences between the JBoss A-MQ xPaaS image and the regular release of JBoss A-MQ, and provides instructions specific to running and configuring the JBoss A-MQ xPaaS image. Documentation for other JBoss A-MQ functionality not specific to the JBoss A-MQ xPaaS image can be found in the https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_A-MQ/[JBoss A-MQ documentation on the Red Hat Customer Portal].
+
+== Differences Between the JBoss A-MQ xPaaS Image and the Regular Release of JBoss A-MQ
+
+There are several major functionality differences in the OpenShift JBoss A-MQ xPaaS image:
+
+* The Karaf shell is not available.
+* The Fuse Management Console (Hawtio) is not available.
+* Configuration of the broker can be performed:
+** using parameters specified in the A-MQ application template, as described in xref:configuring-params[Application Template Parameters].
+** using the STI (Source-to-image) tool, as described in xref:configuring-sti[Configuration Using STI].
+
+
+== Installing the JBoss A-MQ xPaaS Images
+
+To use the Red Hat xPaaS middleware images in your OpenShift project, you must first install the image streams and source-to-image (S2I) application templates.
+
+. In a terminal, change into the root directory of your OpenShift master. It should contain the 'examples' subdirectory.
```

Comment:
yeah this should link to the first steps doc for now, though ultimately this whole situation w/ how templates get installed and updated is going to need to change.